app/src/app/components/contents/downloads/FileRow.tsx

```diff
@@ -0,0 +1,56 @@
+import { ChartColumn, File, Presentation, TableProperties } from "lucide-react";
+import { bytesToSize } from "../../../../helpers";
+import { PacketMetadata } from "../../../../types";
+import DownloadButton from "./DownloadButton";
+
+interface FileRowProps {
+  path: string;
+  packet: PacketMetadata;
+  sharedResource?: boolean;
+}
+
+const presentationExtensions = ["pdf", "html", "ppt", "pptm", "pptx", "potx", "potm", "pps", "xps"];
+const tableExtensions = ["csv", "xls", "xlsx", "xlsm", "xltx", "ods"];
+const imageExtensions = ["jpeg", "jpg", "png", "jiff", "bmp", "gif"];
+const defaultFileIcon = <File className="text-gray-400" />;
+
+const getFileIcon = (extension: string | undefined) => {
+  if (!extension) return defaultFileIcon;
+
+  if (presentationExtensions.includes(extension)) return <Presentation />;
```

Comment:
Inevitably someone's going to have upper case file extensions, so probably should make this case insensitive!